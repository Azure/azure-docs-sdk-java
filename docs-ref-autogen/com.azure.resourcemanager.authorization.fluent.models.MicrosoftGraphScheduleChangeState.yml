### YamlMime:JavaType
uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState"
fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState"
name: "MicrosoftGraphScheduleChangeState"
nameWithType: "MicrosoftGraphScheduleChangeState"
summary: "Defines values for Microsoft<wbr>Graph<wbr>Schedule<wbr>Change<wbr>State."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedClassMethods:
- classRef: "<xref href=\"com.azure.core.util.ExpandableStringEnum?alt=com.azure.core.util.ExpandableStringEnum&text=ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)?alt=com.azure.core.util.ExpandableStringEnum.<T>fromString&text=<T>fromString\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)?alt=com.azure.core.util.ExpandableStringEnum.<T>values&text=<T>values\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)?alt=com.azure.core.util.ExpandableStringEnum.equals&text=equals\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.hashCode()?alt=com.azure.core.util.ExpandableStringEnum.hashCode&text=hashCode\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.toString()?alt=com.azure.core.util.ExpandableStringEnum.toString&text=toString\" data-throw-if-not-resolved=\"False\" />"
- classRef: "java.lang.<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html\">Object</a>"
  methodsRef:
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#clone--\">clone</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#finalize--\">finalize</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#getClass--\">getClass</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notify--\">notify</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notifyAll--\">notifyAll</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait--\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-int-\">wait</a>"
syntax: "public final class MicrosoftGraphScheduleChangeState extends ExpandableStringEnum<MicrosoftGraphScheduleChangeState>"
constructors:
- uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.MicrosoftGraphScheduleChangeState()"
  fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.MicrosoftGraphScheduleChangeState()"
  name: "MicrosoftGraphScheduleChangeState()"
  nameWithType: "MicrosoftGraphScheduleChangeState.MicrosoftGraphScheduleChangeState()"
  syntax: "public MicrosoftGraphScheduleChangeState()"
fields:
- uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.APPROVED"
  fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.APPROVED"
  name: "APPROVED"
  nameWithType: "MicrosoftGraphScheduleChangeState.APPROVED"
  summary: "Static value approved for Microsoft<wbr>Graph<wbr>Schedule<wbr>Change<wbr>State."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState?alt=com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState&text=MicrosoftGraphScheduleChangeState\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final MicrosoftGraphScheduleChangeState APPROVED"
  desc: "Static value approved for MicrosoftGraphScheduleChangeState."
- uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.DECLINED"
  fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.DECLINED"
  name: "DECLINED"
  nameWithType: "MicrosoftGraphScheduleChangeState.DECLINED"
  summary: "Static value declined for Microsoft<wbr>Graph<wbr>Schedule<wbr>Change<wbr>State."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState?alt=com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState&text=MicrosoftGraphScheduleChangeState\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final MicrosoftGraphScheduleChangeState DECLINED"
  desc: "Static value declined for MicrosoftGraphScheduleChangeState."
- uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.PENDING"
  fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.PENDING"
  name: "PENDING"
  nameWithType: "MicrosoftGraphScheduleChangeState.PENDING"
  summary: "Static value pending for Microsoft<wbr>Graph<wbr>Schedule<wbr>Change<wbr>State."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState?alt=com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState&text=MicrosoftGraphScheduleChangeState\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final MicrosoftGraphScheduleChangeState PENDING"
  desc: "Static value pending for MicrosoftGraphScheduleChangeState."
- uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.UNKNOWN_FUTURE_VALUE"
  fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.UNKNOWN_FUTURE_VALUE"
  name: "UNKNOWN_FUTURE_VALUE"
  nameWithType: "MicrosoftGraphScheduleChangeState.UNKNOWN_FUTURE_VALUE"
  summary: "Static value unknown<wbr>Future<wbr>Value for Microsoft<wbr>Graph<wbr>Schedule<wbr>Change<wbr>State."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState?alt=com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState&text=MicrosoftGraphScheduleChangeState\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final MicrosoftGraphScheduleChangeState UNKNOWN_FUTURE_VALUE"
  desc: "Static value unknownFutureValue for MicrosoftGraphScheduleChangeState."
methods:
- uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.fromString(java.lang.String)"
  fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "MicrosoftGraphScheduleChangeState.fromString(String name)"
  summary: "Creates or finds a Microsoft<wbr>Graph<wbr>Schedule<wbr>Change<wbr>State from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for."
    name: "name"
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/String.html\">String</a>"
  syntax: "public static MicrosoftGraphScheduleChangeState fromString(String name)"
  desc: "Creates or finds a MicrosoftGraphScheduleChangeState from its string representation."
  returns:
    description: "the corresponding MicrosoftGraphScheduleChangeState."
    type: "<xref href=\"com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState?alt=com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState&text=MicrosoftGraphScheduleChangeState\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.values()"
  fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.values()"
  name: "values()"
  nameWithType: "MicrosoftGraphScheduleChangeState.values()"
  modifiers:
  - "static"
  syntax: "public static Collection<MicrosoftGraphScheduleChangeState> values()"
  returns:
    description: "known MicrosoftGraphScheduleChangeState values."
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html\">Collection</a>&lt;<xref href=\"com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState?alt=com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState&text=MicrosoftGraphScheduleChangeState\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
desc: "Defines values for MicrosoftGraphScheduleChangeState."
metadata: {}
package: "com.azure.resourcemanager.authorization.fluent.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-authorization:2.19.0
