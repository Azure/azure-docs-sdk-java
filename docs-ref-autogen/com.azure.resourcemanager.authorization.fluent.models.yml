### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.authorization.fluent.models"
fullName: "com.azure.resourcemanager.authorization.fluent.models"
name: "com.azure.resourcemanager.authorization.fluent.models"
summary: "Package containing the inner data models for AuthorizationManagementClient. Role based access control provides you a\n way to apply granular level policy administration down to individual resources or resource groups. These operations\n enable you to manage role definitions and role assignments. A role definition describes the set of actions that can\n be performed on resources. A role assignment grants access to Azure Active Directory users."
classes:
- "com.azure.resourcemanager.authorization.fluent.models.ADGroupInner"
- "com.azure.resourcemanager.authorization.fluent.models.ApplicationInner"
- "com.azure.resourcemanager.authorization.fluent.models.CheckGroupMembershipResultInner"
- "com.azure.resourcemanager.authorization.fluent.models.ClassicAdministratorInner"
- "com.azure.resourcemanager.authorization.fluent.models.DirectoryObjectInner"
- "com.azure.resourcemanager.authorization.fluent.models.DomainInner"
- "com.azure.resourcemanager.authorization.fluent.models.KeyCredentialInner"
- "com.azure.resourcemanager.authorization.fluent.models.OAuth2PermissionGrantInner"
- "com.azure.resourcemanager.authorization.fluent.models.PasswordCredentialInner"
- "com.azure.resourcemanager.authorization.fluent.models.PermissionInner"
- "com.azure.resourcemanager.authorization.fluent.models.ProviderOperationsMetadataInner"
- "com.azure.resourcemanager.authorization.fluent.models.RoleAssignmentInner"
- "com.azure.resourcemanager.authorization.fluent.models.RoleDefinitionInner"
- "com.azure.resourcemanager.authorization.fluent.models.ServicePrincipalInner"
- "com.azure.resourcemanager.authorization.fluent.models.ServicePrincipalObjectResultInner"
- "com.azure.resourcemanager.authorization.fluent.models.UserInner"
metadata: {}
package: "com.azure.resourcemanager.authorization.fluent.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-authorization:2.1.0
