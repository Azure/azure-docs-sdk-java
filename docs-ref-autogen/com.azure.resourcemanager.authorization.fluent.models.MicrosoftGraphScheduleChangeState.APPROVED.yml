### YamlMime:JavaMember
uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.APPROVED*"
fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.APPROVED"
name: "APPROVED"
nameWithType: "MicrosoftGraphScheduleChangeState.APPROVED"
members:
- uid: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.APPROVED"
  fullName: "com.azure.resourcemanager.authorization.fluent.models.MicrosoftGraphScheduleChangeState.APPROVED"
  name: "APPROVED"
  nameWithType: "MicrosoftGraphScheduleChangeState.APPROVED"
  summary: "Static value approved for MicrosoftGraphScheduleChangeState."
  syntax: "public static final MicrosoftGraphScheduleChangeState APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.authorization.fluent.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-authorization:2.3.0
